lib/ex_toolkit/roman.ex

```diff
@@ -0,0 +1,20 @@
+defmodule ExToolkit.Roman do
+  @valid_romans ["I", "V", "X", "L", "C", "D", "M"]
+
+  def is_valid_roman(string) do
+    string
+    |> String.upcase()
+    |> String.graphemes()
+    |> is_valid_roman(@valid_romans)
```

Comment:
This does not make sense since you are using directly `@valid_romans` in the bottom case. You just need a guard.